// @lolikillers/aexm-api/index.js
const { 
	otakudesu,
	covid,
	delta,
	radio1,
	radio2,
	ongoing,
	komiku,
	joox,
	tebakgambar,
	surah,
	sholat,
	lirik,
	chara,
	wattpad,
	playstore,
	linkwa,
	pinterest,
	igdl,
	igstory,
	igstalk,
	twitter,
	fbdown,
	youtube,
	ttdownloader
} = require('./lib/function')

module.exports.otakudesu = otakudesu
module.exports.covid = covid
module.exports.delta = delta
module.exports.radio1 = radio1
module.exports.radio2 = radio2
module.exports.ongoing = ongoing
module.exports.komiku = komiku
module.export.joox = joox
module.exports.tebakgambar = tebakgambar
module.exports.surah = surah
module.exports.sholat = sholat
module.exports.lirik = lirik
module.exports.chara = chara
module.exports.wattpad = wattpad
module.exports.playstore = playstore
module.exports.linkwa = linkwa
module.exports.pinterest = pinterest
module.exports.igdl = igdl
module.exports.igstory = igstory
module.exports.igstalk = igstalk
module.exports.twitter = twitter
module.exports.fbdown = fbdown
module.exports.youtube = youtube
module.exports.ttdownloader = ttdownloader
